app/react/Forms/components/Geolocation.js

```diff
@@ -61,32 +85,43 @@ export default class Geolocation extends Component {
   render() {
     const markers = [];
     const { value: [value] } = this.props;
+    const latitude = value ? value.lat : '';
+    const longitude = value ? value.lon : '';
```

Comment:
This shouldn't be needed.  There is a defaultProps defined on the component for this exact scenario.  What is triggering this to not have the correct values?

Reply:
Now the code is a little bit different, now is used always the getInputValues method to get the props